code/modules/paperwork/paper.dm

```diff
@@ -484,6 +484,14 @@
 /obj/item/paper/crumpled/bloody
 	icon_state = "scrap_bloodied"
 
+//////lavaland clown planet papers
+
+/obj/item/paper/crumpled/bloody/ruins/lavaland/clown_planet/escape
```

Comment:
I personally think it's better to put these in their own clown_planet file in the code\modules\ruins\lavalandruin_code folder.